packages/components/color-picker/src/index.vue

```diff
@@ -211,9 +211,15 @@ export default defineComponent({
     watch(
       () => currentColor.value,
       (val) => {
+        const currentValueColor = new Color({
+          enableAlpha: props.showAlpha,
+          format: props.colorFormat,
+        })
```

Comment:
I think this is a problem when `new Color` is initialized
```javascript
new Color({
        enableAlpha: props.showAlpha,
        format: props.colorFormat,
        value: props.modelValue,
})
```
and `Color` constructor change to 
```typescript
  constructor(options?: Options) {
    options = options || ({} as Options)

    for (const option in options) {
      if (hasOwn(options, option)) {
        this[option] = options[option]
      }
    }
    if (options.value) {
      this.fromString(options.value)
    } else {
      this.doOnChange()
    }
  }
```